backend/api/Program.cs

```diff
@@ -41,6 +41,10 @@
     }
 }
 
+bool useInMemoryDatabase = builder.Configuration
+    .GetSection("Database")
+    .GetValue<bool>("UseInMemoryDatabase");
+
```

Comment:
Move this to just before it is used (line 68)

Reply:
Fixed